### PR TITLE
BS-14464 As dave I want to access groovy documentation when previewing

### PR DESCRIPTION
**Pull request by azara**
initialization script

BS-14464 As dave I want to access groovy documentation when previewing
initialization script

-the next button is rename "preview" when user is on attribute selection page
-a link to groovy script documentation has been added
-when previewing the script and clicking ok, if user cancels pop up, he
will be rederacted to previous page and generate manually will be
selected

### CODE REVIEW

File: bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/ui/wizard/AddInputContractFromDataWizardDialog.java [hunk 1]
```diff
@@ -56,4 +57,22 @@ protected void actionOnFinishAndAdd() {
...
+    /*
+     * (non-Javadoc)
+     * @see org.bonitasoft.studio.common.jface.dialog.FinishAndAddCustomWizardDialog#updateButtons()
+     */
+    @Override
+    public void updateButtons() {
+        super.updateButtons();
+        final Button nextButton = getButton(IDialogConstants.NEXT_ID);
+        if (getCurrentPage() instanceof CreateContractInputFromBusinessObjectWizardPage) {
+            final CreateContractInputFromBusinessObjectWizardPage page = (CreateContractInputFromBusinessObjectWizardPage) getCurrentPage();
```
Comment (rbioteau) [inline bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/ui/wizard/AddInputContractFromDataWizardDialog.java]: page variable is not used.

File: bundles/plugins/org.bonitasoft.studio.contract/messages.properties [hunk 2]
```diff
@@ -79,3 +79,4 @@ generatedScriptPreviewTitle=Generated script preview
 scriptNameLabel=Script name
 selectAll=Select all
 deselectAll=Deselect all
+preview=Preview
```
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.contract/messages.properties]: I think we should keep the > as we stay in the same wizard.

File: bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/ui/wizard/AddInputContractFromDataWizardDialog.java (hunk 1 above)
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/ui/wizard/AddInputContractFromDataWizardDialog.java]: this line seem useless

**Comment by apupier** [comment]
in commit it is stated that 'when previewing the script and clicking ok, if user cancels pop up, he
will be rederacted to previous page and generate manually will be
selected" but  the wizard just exits

**Pull request merged**